src/login/Login.jsx

```diff
@@ -93,40 +100,31 @@ export default function Login() {
                                 />
                             </p>
                         </form-group>
-                        <form-group>
-                            <input type="checkbox" className="my-2" name="show_password_checkbox" id="showPassword" onClick={handleTogglePasswordDisplay}/>
-                            <label className="ml-2 my-2" htmlFor="showPassword">Show Password</label>
-                        </form-group>
-                        <div><br></br></div>
+                       
                         <div>
                             {errorMessage && <span className="error" name="errorMessage" aria-label="errorMessage" role="alert">{errorMessage}</span>}
                         </div>
+                        
                         <div className="row">
-                            <label>Not yet registered? Click on sign up.</label>
+                            <label>Dont have an Account? Sign Up.</label>
```

Comment:
Link this to ```/register```

Reply:
Yes Sure @stableapple 
Also can you please tell me how to remove package.json?

Reply:
Can you please tell me how to do I commit latest code to this pull request only ?

Reply:
To remove the ```package-lock.json``` you can do ```git reset --soft HEAD~1``` and then ```git rm package-lock.json```. To resolve the conflict, that is to merge the latest code from the master branch(develop) to your new branch that you are currently working in, you will have to change to the new branch by doing ```git checkout new_branch```. Then set upstream by ``` git remote add upstream https://github.com/anitab-org/bridge-in-tech-web.git``` followed by ```git pull upstream develop```

Reply:
Thanks a lot 

git pull upstream develop
From https://github.com/anitab-org/bridge-in-tech-web
 * branch            develop    -> FETCH_HEAD
Already up to date.

This is what I am getting